spec/controllers/case_contacts_controller_spec.rb

```diff
@@ -25,6 +25,45 @@
     allow(controller).to receive(:current_user).and_return(volunteer)
   end
 
+  describe "GET index" do
```

Comment:
I used these tests to identify exactly where the bug was coming from, so they're scoped broadly. Only the "returns filtered case contacts by contact type" test is truly required to verify behavior here, BUT the `index` route wasn't tested at all so I left the bit of extra coverage for good measure. 🔮 🤞